docs/en/cli.md

```diff
@@ -0,0 +1,475 @@
+  test1 13 0 172.27.128.33:8541 follower 0min yes kNoCompress 0 0 0.000
+  test1 13 1 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 2 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 3 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+> addreplica test1 1,2,3 172.27.128.33:8541
+AddReplica ok
+> addreplica test1 1-3 172.27.128.33:8541
+AddReplica ok
+```
+
+#### delreplica
+
+Delete replica
+
+Command format: `delreplica table_name pid_group endpoint`
+
+* table\_name table name
+* pid\_group shard id collection. There can be the following situations
+    * a single shard
+    * Multiple shards, separated by commas. Such as 1,3,5
+    * Shard interval, the interval is a closed interval. For example, 1-5 means shard 1, 2, 3, 4, 5
+* endpoint endpoint to delete replicas
+
+```
+> showtable test1
+  name tid pid endpoint role ttl is_alive compress_type offset record_cnt memused
+-------------------------------------------------- -------------------------------------------------- --------
+  test1 13 0 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 0 172.27.128.33:8541 follower 0min yes kNoCompress 0 0 0.000
+  test1 13 1 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 1 172.27.128.33:8541 follower 0min yes kNoCompress 0 0 0.000
+  test1 13 2 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 2 172.27.128.33:8541 follower 0min yes kNoCompress 0 0 0.000
+  test1 13 3 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 3 172.27.128.33:8541 follower 0min yes kNoCompress 0 0 0.000
+> delreplica test1 0 172.27.128.33:8541
+DelReplica ok
+> showtable test1
+  name tid pid endpoint role ttl is_alive compress_type offset record_cnt memused
+-------------------------------------------------- -------------------------------------------------- --------
+  test1 13 0 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 1 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 1 172.27.128.33:8541 follower 0min yes kNoCompress 0 0 0.000
+  test1 13 2 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 2 172.27.128.33:8541 follower 0min yes kNoCompress 0 0 0.000
+  test1 13 3 172.27.128.31:8541 leader 0min yes kNoCompress 0 0 0.000
+  test1 13 3 172.27.128.33:8541 follower 0min yes kNoCompress 0 0 0.000
+> delreplica test1 1,2,3 172.27.128.33:8541
+DelReplica ok
+> delreplica test1 1-3 172.27.128.33:8541
+DelReplica ok
+```
+
+#### migrate
+
+copy migration
+
+Command format: `migrate src_endpoint table_name pid_group des_endpoint`
+
+* src\_endpoint The node that needs to be checked out
+* table\_name table name
+* pid\_group shard id collection. There can be the following situations
+    * a single shard
+    * Multiple shards, separated by commas. Such as 1,3,5
+    * Shard interval, the interval is a closed interval. For example, 1-5 means shard 1, 2, 3, 4, 5
```

Comment:
please replace those description as the same as the that of addreplica

Reply:
Ok, thank you.